src/_common/events/event-bus.service.ts

```diff
@@ -0,0 +1,33 @@
+import Vue from 'vue';
+
+export type EventBusDeregister = () => void;
+
+export class EventBus {
+	private e!: Vue;
```

Comment:
Please call this something like "element", or even "elem".
With "e" I need to figure out what this even is.

Reply:
its self contained within the event bus, i don't see why make it super verbose when its one "goto definition" away.